_extras/guide.md

```diff
@@ -173,6 +173,10 @@ particular set of files in `.gitignore`.
 *   It is very useful to draw a diagram showing the different repositories
     involved.
 
+*   Learners using syntax from GitHub, `git push -u origin master`, will have slightly different 
```

Comment:
I like this addition to the guide, but it might be helpful if there were slightly more context. Maybe something like:

> When pushing to a remote, the output from Git can vary slightly depending on what leaners execute. The lesson displays the output from git if a learner executes `git push origin master`. However, some learners might use syntax suggested by GitHub for pushing to a remote with an existing repository, which is `git push -u origin master` ... 

and then include everything you've put! What do you think?

Reply:
Sounds good!

Reply:
I updated the instructor guide to add your suggestion.